modules.d/90crypt/module-setup.sh

```diff
@@ -66,6 +66,8 @@ install() {
         inst_hook cleanup 30 "$moddir/crypt-cleanup.sh"
     fi
 
+    # Have to use exit here, return value gets ignored in dracut.sh...
+    uuid_map_file=$(mktemp /tmp/dracut.XXXXXXXXXX) || exit 1
```

Comment:
Good question. I guess the original author wanted to install the file "properly". Want me to make the change?